cmd/buildah/main.go

```diff
@@ -68,6 +68,14 @@ func main() {
 	}
 	app.Commands = []cli.Command{
 		{
+			Name:        "images",
```

Comment:
We're already using "list" for listing containers that buildah is working with, though I could be swayed to "list-images" to parallel with "delete-image".